tests/lib/test-crio.bats.in

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env bats
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# CC tests data
+TEST_DATA="${BATS_TEST_DIRNAME}/../../data/"
+
+# CRI-O URL and version
+CRIO_GITHUB="github.com/kubernetes-incubator/cri-o"
+CRIO_VERSION="3195f459"
+
+# CRI-O root directory
+OCID_ROOT="$GOPATH/src/$CRIO_GITHUB"
+
+# Path of the ocid binary.
+OCID_BINARY=${OCID_BINARY:-${OCID_ROOT}/ocid}
+# Path of the ocic binary.
+OCIC_BINARY=${OCIC_BINARY:-${OCID_ROOT}/ocic}
+# Path of the conmon binary.
+CONMON_BINARY=${CONMON_BINARY:-${OCID_ROOT}/conmon/conmon}
+# Path of the pause binary.
+PAUSE_BINARY=${PAUSE_BINARY:-${OCID_ROOT}/pause/pause}
+# Path of the default seccomp profile.
+SECCOMP_PROFILE=${SECCOMP_PROFILE:-${OCID_ROOT}/seccomp.json}
+# Name of the default apparmor profile.
+APPARMOR_PROFILE=${APPARMOR_PROFILE:-ocid-default}
+# Path of the bin2img binary.
+BIN2IMG_BINARY=${BIN2IMG_BINARY:-${OCID_ROOT}/test/bin2img/bin2img}
+# Path of the copyimg binary.
+COPYIMG_BINARY=${COPYIMG_BINARY:-${OCID_ROOT}/test/copyimg/copyimg}
+# Path of tests artifacts.
+ARTIFACTS_PATH="@CRIO_CACHE@"
+# Path of the checkseccomp binary.
+CHECKSECCOMP_BINARY=${CHECKSECCOMP_BINARY:-${OCID_ROOT}/test/checkseccomp/checkseccomp}
+
+# CRI-O Runtime
+RUNTIME=${RUNTIME:-cc-oci-runtime}
+RUNTIME_PATH=$(command -v $RUNTIME || true)
+RUNTIME_BINARY=${RUNTIME_PATH:-/usr/bin/cc-oci-runtime}
```

Comment:
Is this dash expected ? Should not it be ${RUNTIME_PATH:/usr/bin/cc-oci-runtime} instead ?